sdk/identity/azure-identity/pom.xml

```diff
@@ -27,7 +27,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.6.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.7.0-beta.3</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```

Comment:
Does this need to be current?

Reply:
Not sure what's the right thing here to do here. Vinay says this should be unreleased. But current seems to be able to pass pipeline builds as well.

Reply:
Only items within the same sdk/<area> use "current". If you need a source built binary from another pipeline then the correct way to do that is to create an unreleased_ entry and it's version gets set to the 'current' version. What @jianghaolu did is correct.

Reply:
Thanks for the clarification!